app/auth.py

```diff
@@ -0,0 +1,36 @@
+from flask import jsonify, g, current_app, request
+from .models import User
+from .models import db
+from app import app, auth
+
+
+db.create_all()
+
+
+@app.route("/auth/register", methods=["POST"])
+def register():
+    username = request.json.get("username")
+    password = request.json.get("password")
+    if username and password:
```

Comment:
What if someone does not pass both **username** and **password** during registration?

Consider :

```
if not username and not password
    return jsonify({"message":'Both username and password are required to sign up'})
```